server/src/main/java/org/opensearch/node/NodeResourceUsageStats.java

```diff
@@ -54,6 +64,7 @@ public String toString() {
         sb.append("Timestamp: ").append(timestamp);
         sb.append(", CPU utilization percent: ").append(String.format(Locale.ROOT, "%.1f", cpuUtilizationPercent));
         sb.append(", Memory utilization percent: ").append(String.format(Locale.ROOT, "%.1f", memoryUtilizationPercent));
+        sb.append(this.ioUsageStats.toString());
```

Comment:
We haven't overriden toString() in IoUsageStats , what will be the result ?

Reply:
I have overriden toString Method in `IoUsageStats` class.